cmd/snap-bootstrap/partition/encrypt.go

```diff
@@ -56,8 +56,6 @@ func NewEncryptionKey() (EncryptionKey, error) {
 
 // Store writes the LUKS key in the location specified by filename.
 func (key EncryptionKey) Store(filename string) error {
-	// TODO:UC20: provision the TPM, generate and store the lockout authorization,
-	//            and seal the key. Currently we're just storing the unprocessed data.
 	if err := ioutil.WriteFile(filename, key[:], 0600); err != nil {
```

Comment:
Should we use osutil.AtomicWriteFile() here?

Reply:
Like we did with the lockout authorization file, yes.

Reply:
In fact we don't store it unsealed anymore, this function is now unnecessary and will be removed.